src/main/archetype/ui.content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -1,10 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
+    <filter root="/conf" mode="merge"/>
```

Comment:
These should go to the `ui.apps.structure` package (per https://docs.adobe.com/content/help/en/experience-manager-cloud-service/implementing/developing/repository-structure-package.html)

Reply:
moving it there gives results in another warning, since the structure package becomes mixed:

jackrabbit-packagetype: Package of type 'MIXED' is legacy. Use one of the other types instead! (org.apache.jackrabbit:filevault-package-maven-plugin:1.1.4:validate-files:default-validate-files:process-classes)	properties.xml	/testing-basic.ui.apps.structure/target/vault-work/META-INF/vault	line 0	Maven Build Participant Problem

Apparently it should only contain /apps stuff